.github/labeler.yml

```diff
@@ -0,0 +1,22 @@
+"@conf":
+  - changed-files:
+    - any-glob-to-any-file:
+      - .github/**
+      - "*"
+      - "**/.github/**"
```

Comment:
`- "**/.github/**"` matches any .github directory at any depth. Will it makes `- .github/**` redundant?